readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: PostNL, shadim, abdalsalaam
 Tags: woocommerce, export, delivery, packages, PostNL, Shipping
 Requires at least: 4.6
 Requires PHP: 5.6
-Tested up to: 6.6
+Tested up to: 6.7
 Stable tag: 5.6.2
 WC requires at least: 4.0
 WC tested up to: 9.3
```

Comment:
please bump this to 9.4 since its released and we already tested it.

Reply:
@Abdalsalaam updated